package.json

```diff
@@ -21,6 +21,11 @@
     "url": "https://github.com/Sage/carbon/issues"
   },
   "homepage": "https://github.com/Sage/carbon#readme",
+  "peerDependencies": {
+    "flux": "~2.1.1",
+    "react": "~15.6.1",
```

Comment:
Could these be dependencies? While it would remove any flexibility for consumers to use different versions of React, we wouldn't really support any other version of React (eg they couldn't use v16 if we didn't yet support it).

It would also mean that if we control the version we could ensure that applications remain up to date with minor versions so consumers don't have to worry about it so much.

On the other side, if we controlled the version we're not guaranteeing that consumers will remain up to date with Carbon - so if they are using a version of Carbon locked to version 15.6.0 and they need to use 15.7.0, they may need to do a lot of upgrading of Carbon versions (which may or may not be a bad thing).

Peer dependency is useful for modules which are supplying a small piece to an application, whereas Carbon is more of a library for building and managing your application which is why I don't see declaring them as dependencies as such a negative issue.

Interested to hear opinions on what people would prefer.